ui/src/timeMachine/components/view_options/YAxisBase.tsx

```diff
@@ -23,34 +23,34 @@ class YAxisBase extends PureComponent<Props> {
 
     return (
       <Grid.Column widthXS={Columns.Twelve}>
-        <Form.Element label="Y-Value's Format">
+        <Form.Element label="Y-Value Unit Prefix">
           <Radio shape={ButtonShape.StretchToFit}>
             <Radio.Button
               id="y-values-format-tab--raw"
               value=""
               active={base === ''}
-              titleText="Don't format values"
+              titleText="Do not format values using a unit prefix"
               onClick={onUpdateYAxisBase}
             >
-              Raw
+              None
             </Radio.Button>
             <Radio.Button
               id="y-values-format-tab--kmb"
               value={BASE_10}
               active={base === BASE_10}
-              titleText="Thousand / Million / Billion"
+              titleText="Format values using a SI unit prefix"
```

Comment:
What do you think of spelling out the full name of `SI` in the `titleText`?

Reply:
Good point! I updated it.